code-of-conduct.md

```diff
@@ -82,20 +82,20 @@ A direct message ("DM") is a private message to one or more other members. Sendi
 
 ## Not For Profit (Mostly)
 
-This community is hyper-allergic to unwanted commercial behavior. This is not a place for prominent commercial activities such as recruiting, lead generation, marketing, and other solicitation, except in channels dedicated to that purpose. This includes starting market research conversations outside of #market-research. If you join this community to take value rather than contribute, the community will quickly notice and react.
+Some parts of this community are averse to unwanted commercial behavior. This is not the best place for prominent commercial outreach activities such as recruiting, lead generation, marketing, and other solicitation, except in channels dedicated to that purpose. This includes performing unambiguously commercial market research conversations for paid products outside of #market-research. If it's unclear: ask yourself whether the conversation will at least as valuable to other members of the community than it is for you. If you join this community primarily to take value rather than make balanced contributions, the community will quickly notice and react.
 
 While apparent commercial behavior is easy to identify, non-obvious commercial behavior is a grey area. In general, if you are wondering if a post is commercially appropriate, our guidelines are that you:
 
 * Ask permission from the channel before posting a message that might be considered commercial, stating clear intent for the post, and, 
-* Listen to the response of your peers in the channel and act accordingly.
+* Evaluate the majority response of your peers in the channel and act accordingly.
```

Comment:
Members should try to understand the message in the responses, not count the number of responders -- it's not a vote.

Reply:
@wlonkly What I'm trying to combat here is when 10 people go "yeh this seems fine" and one person says "this is market research" and asks for it to get deleted (and it gets deleted). I've seen this happen before and it seems non-ideal. I welcome alternate wording to make this intention clearer.